tests/common/global_psp_checks.go

```diff
@@ -0,0 +1,50 @@
+package common
+
+import (
+	"github.com/rancher/hull/pkg/checker"
+	"github.com/rancher/hull/pkg/test"
+	"github.com/stretchr/testify/assert"
+	policyv1 "k8s.io/api/policy/v1beta1"
+	rbacv1 "k8s.io/kubernetes/pkg/apis/rbac"
+)
+
+func GetPspEnabledCheck(numPspsWithPspEnabled int) test.Checks {
```

Comment:
PSP should be capitalized?

Reply:
Can we rename this check? I'd expect it to be something like`EnsurePSPsExist(numExpectedPSPs int)`